### DISCUSSION TITLE
Should you wrap all screens, or is it sufficient to wrap only the screens that use react-native-gesture-handler?

### DISCUSSION THREAD

**Discussion by rendomnet** [software-mansion/react-native-gesture-handler#3261] [Q&A]
I’m using react-native-navigation; do I need to wrap all my screens with react-native-gesture-handler? I only need it for ‘React Native Reanimated Carousel.’ Is it sufficient to wrap just the components containing the carousel?

**Comment by Glazzes** [comment]
Ideally you should only wrap the entry point of your app (App.tsx file) and modal screens.
